packaging/nuget/nservicebus.transporttests.nuspec

```diff
@@ -27,6 +27,6 @@
     <file
       src="..\..\src\NServiceBus.TransportTests\**\*.cs"
       target="content\App_Packages\NSB.TransportTests.$version$"
-      exclude="**\bin\**\*.*;**\obj\**\*.*" />
+      exclude="**\bin\**\*.*;**\obj\**\*.*;..\..\src\NServiceBus.TransportTests\ConfigureMsmqTransportInfrastructure.cs" />
```

Comment:
@timbussmann I guess we could keep it for reference but TBH I did not see it being much of a value when I was working with transport maintainers to set the test up